api/renderers.py

```diff
@@ -0,0 +1,23 @@
+import vobject
```

Comment:
If you add `"vobject"` and `"twilio.request_validator"` to the ignored modules list, `mypy` will like your PR.